src/app/client/src/app/modules/shared/services/config/url.config.json

```diff
@@ -225,7 +225,8 @@
       },
       "DELETE_USER": "user/v1/block",
       "UNDELETE_USER": "user/v1/unblock",
-      "UPDATE_USER_ORG_ROLES": "user/v2/role/assign"
+      "UPDATE_USER_ORG_ROLES": "user/v1/role/assign",
+      "UPDATE_USER_ORG": "user/v2/role/assign"
```

Comment:
@snehalsabade Why these 2 APIs are diffrent?

Reply:
- user/v1/role/assign is for role update 
- user/v2/role/assign is for organisation update

Reply:
@snehalsabade can we use only one for both action? like user/v2/role/assign

Reply:
No we can not use same user/v2/role/assign for both, because there are two different request.

Reply:
Please merge this PR for notification reverted changes ,I will update API changes after getting response in discussion forum.

Reply:
Add Ticket link here @snehalsabade

Reply:
[nulp_creator5  / password](https://github.com/orgs/Sunbird-Lern/discussions/164#discussioncomment-7238384)

Reply:
https://github.com/orgs/Sunbird-Lern/discussions/164#discussioncomment-7238384